masq_lib/src/messages.rs

```diff
@@ -843,6 +845,12 @@ pub struct UiWalletAddressesResponse {
 }
 conversation_message!(UiWalletAddressesResponse, "walletAddresses");
 
+#[derive(Message, Clone, PartialEq, Eq)]
+pub struct MessageScheduler<M: Message> {
+    pub schedule_msg: M,
+    pub duration: Duration,
```

Comment:
Instead of repeating the word duration, the filed could say something like `msg_delay`

Reply:
Instead, I just kept it to `delay`.